deepcell/applications/pannuke_nuclear_seg.py

```diff
@@ -0,0 +1,127 @@
+        'n_epochs': 5,
+        'training_steps_per_epoch': 1987,
+        'validation_steps_per_epoch': 253
+    }
+
+    def __init__(self, model=None):
+
+        if model is None:
+            archive_path = tf.keras.utils.get_file(
+                'NuclearSegmentation.tgz', MODEL_PATH,
+                file_hash='7fff56a59f453252f24967cfe1813abd',
+                extract=True, cache_subdir='models'
+            )
+            model_path = os.path.splitext(archive_path)[0]
+            model = tf.keras.models.load_model(model_path)
+
+        super(NuclearSegmentation, self).__init__(
+            model,
+            model_image_shape=model.input_shape[1:],
+            model_mpp=0.65,
+            preprocessing_fn=normalize,
+            postprocessing_fn=deep_watershed,
+            dataset_metadata=self.dataset_metadata,
+            model_metadata=self.model_metadata)
+
+    def predict(self,
+                image,
+                batch_size=4,
+                image_mpp=None,
+                pad_mode='reflect',
+                preprocess_kwargs=None,
+                postprocess_kwargs=None):
+        """Generates a labeled image of the input running prediction with
+        appropriate pre and post processing functions.
+        Input images are required to have 4 dimensions
+        ``[batch, x, y, channel]``.
+        Additional empty dimensions can be added using ``np.expand_dims``.
+        Args:
+            image (numpy.array): Input image with shape
+                ``[batch, x, y, channel]``.
+            batch_size (int): Number of images to predict on per batch.
+            image_mpp (float): Microns per pixel for ``image``.
+            pad_mode (str): The padding mode, one of "constant" or "reflect".
+            preprocess_kwargs (dict): Keyword arguments to pass to the
+                pre-processing function.
+            postprocess_kwargs (dict): Keyword arguments to pass to the
+                post-processing function.
+        Raises:
+            ValueError: Input data must match required rank of the application,
+                calculated as one dimension more (batch dimension) than expected
+                by the model.
+            ValueError: Input data must match required number of channels.
+        Returns:
+            numpy.array: Labeled image
+        """
+        if preprocess_kwargs is None:
+            preprocess_kwargs = {}
+
+        if postprocess_kwargs is None:
+            postprocess_kwargs = {
+                'min_distance': 10,
+                'detection_threshold': 0.1,
+                'distance_threshold': 0.01,
+                'exclude_border': False,
+                'small_objects_threshold': 0
+            }
+
+        return self._predict_segmentation(
+            image,
+            batch_size=batch_size,
+            image_mpp=image_mpp,
+            pad_mode=pad_mode,
+            preprocess_kwargs=preprocess_kwargs,
+            postprocess_kwargs=postprocess_kwargs)
```

Comment:
```suggestion
            postprocess_kwargs=postprocess_kwargs)

```

Add a newline to the end of the file.